backend/app/profiles.py

```
@@ -243,6 +243,84 @@ async def create_profile(profile_data: ProfileCreate, current_user = Depends(get
         documents=[{"id": d.id, "url": d.url, "type": d.type} for d in profile.documents]
     )
 
+@router.put("/me", response_model=ProfileResponse)
+async def update_my_profile(profile_data: ProfileCreate, current_user = Depends(get_current_user)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)